packages/react-lowcode/src/codegen/generation/generators/index/basic-table-generator.ts

```diff
@@ -4,18 +4,20 @@ import { Entity, Property } from '../../entity/index'
 import { TableGenerator } from './table-generator-factory'
 import { TableComponentDefinition } from '../../../definition/table-definition-core'
 import { MuiTableComponents } from '../../../definition/material-ui/table'
+import { GrommetTableComponents } from '../../../definition/grommet/table'
 import TableGeneratorBase from './table-generator-base'
 import GenerationContext from "../../context"
+import { Formatter, UiFramework } from "../../../definition/context-types"
 
 export class BasicTableGenerator extends TableGeneratorBase  implements TableGenerator
 {
-    constructor(generationContext: GenerationContext) {
-        super(generationContext);
+    constructor(generationContext: GenerationContext, entity: Entity) {
```

Comment:
Let's not add new constructor attribute. I would rather add new parameter to `generateTableComponent`. We would split the original context to options  (suitable for constructor) and stateless params during code generation.